scene/animation/tween.cpp

```diff
@@ -1175,17 +1249,146 @@ bool Tween::interpolate_deferred_callback(Object *p_object, real_t p_duration, S
 	data.arg[3] = p_arg4;
 	data.arg[4] = p_arg5;
 
-	pending_update++;
-	interpolates.push_back(data);
-	pending_update--;
-	return true;
+	if (pending_update) {
+		interpolates_queued.push_back(data);
+		_add_pending_command("interpolate_queued", data.handle);
+	} else {
+		pending_update++;
+		interpolates.push_back(data);
+		pending_update--;
+	}
+
+	return data.handle;
 }
 
-bool Tween::follow_property(Object *p_object, NodePath p_property, Variant p_initial_val, Object *p_target, NodePath p_target_property, real_t p_duration, TransitionType p_trans_type, EaseType p_ease_type, real_t p_delay) {
-	if (pending_update != 0) {
-		_add_pending_command("follow_property", p_object, p_property, p_initial_val, p_target, p_target_property, p_duration, p_trans_type, p_ease_type, p_delay);
-		return true;
+Tween::HANDLE Tween::create_sequence(Tween::HANDLE h0, Tween::HANDLE h1, Tween::HANDLE h2, Tween::HANDLE h3, Tween::HANDLE h4, Tween::HANDLE h5, Tween::HANDLE h6, Tween::HANDLE h7, Tween::HANDLE h8, Tween::HANDLE h9, Tween::HANDLE h10, Tween::HANDLE h11, Tween::HANDLE h12) {
```

Comment:
I think you should use `VARIANT_ARG_DECLARE` and mark this function as a vararg. This seems quite clumsy to maintain.

Reply:
That's definitely one of the things I will consider cleaning up in case the overall proposal is accepted - I did this way only because it was faster. I would also like to have HANDLE appear in the method tooltip/documentation as right now you see a bunch of `int`s which is quite confusing.